docs/service-patterns/apply.md

```diff
@@ -0,0 +1,159 @@
+- I need to accurately capture applicant information so that there are no mistakes.
+- I need to ensure I only ask relevant questions to applicants so that I can collect the right information.
+- I need a clear list of supporting evidence (where necessary) so that I can easily validate and know information is correct.
+- I need to make sure sensitive information is only seen by relevant people so that the department meets GDPR standards.
+- I need to know that the information we hold about an applicant is up-to-date so that I can advise on next steps appropriately.
+- I need an efficient way of communicating personal data changes across products so that it saves time.
+- I need to be confident the data I'm accessing is accurate and up-to-date.
+- I need to have confidence that users have access to relevant guidance to make a decision because I cannot advise them on their specific circumstances.
+
+{% endtab %}
+{% tab "Stages and steps" %}
+
+## Stages and steps  
+
+### User scenarios
+
+These are some scenarios you may have to design for as part of your service.
+
+Happy paths:
+
+- Individual applying for themselves
+- Individual applying on behalf of someone else
+- Business or organisation applying for themselves
+- Business or organisation applying on behalf of someone else
+- Staff user applying for something on behalf of an individual, business or organisation
+- Multiple user types contributing to the same application
+
+Variations:
+
+- Additional application information is needed before it can be processed
+- Evidence in support of application needs to be resent because it's deemed unsuitable
+- User decides they want to withdraw their application
+- User information changes between starting application and completion
+- Eligibility to apply changes while applying
+- User pauses application and returns to it
+- User switches channel during application
+- On checking their answers, user changes something which significantly affects subsequent answers
+- User requires point of contact for support
+
+Unhappy Paths:
+
+- User does not understand what they need to complete application
+- User does not meet the eligibility threshold or criteria
+- User cannot find the application form
+- Application is missing information or contains the wrong information
+- User did not receive receipt notification
+- Third party does not complete their part of the application
+- Time limit elapse (for example, SLA for response)
+- Application window closes
+- User is logged out of the service part way through their application due to time out
+
+{% endtab %}
+{% tab "Contribute" %}
+
+## Contributing to MoJ service patterns
+
+You'll likely design multiple instances of potential service patterns when you develop new products or services.
+
+Contact <a href="mailto:martin.ford-downes@digital.justice.gov.uk">Martin Ford-Downes</a> if you have a potential service pattern idea or want to help us work on others.
```

Comment:
At some point Martin won't have an @digital.justice.gov.uk email address because of the Google decommissioning work.  Should we change to @justice.gov.uk?

Reply:
It'll automatically forward through to the new email address so we don't need to worry about updating it.

Also, if this prototype by Martin and others working on service patterns works well, there'll need to be a more formal contact route which would remove the need for Martin's personal email address being used.